plugins/vi-mode/vi-mode.plugin.zsh

```diff
@@ -50,13 +50,16 @@ zle -N zle-keymap-select
 # These "echoti" statements were originally set in lib/key-bindings.zsh
 # Not sure the best way to extend without overriding.
 function zle-line-init() {
+  LAST_VI_KEYMAP=$VI_KEYMAP
   typeset -g VI_KEYMAP=main
+  [[ $LAST_VI_KEYMAP != 'main' ]] && [[ "${VI_MODE_RESET_PROMPT_ON_MODE_CHANGE:-}" = true ]] && zle reset-prompt
```

Comment:
```suggestion
  [[ "${prev_vi_keymap}" != 'main' ]] && [[ "${VI_MODE_RESET_PROMPT_ON_MODE_CHANGE:-}" = true ]] && zle reset-prompt
```
Just a quoting suggestion and using the local-var name from suggestion above

Reply:
Yep, included these in the commit that addresses the above.